### PR TITLE
Nathaniel - ETA API

### PR DESCRIPTION
**Pull request by rodrigotiscareno**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# victini-microservices
\ No newline at end of file
+# victini-microservices
+eta Flask API integration
```
Comment (rodrigotiscareno) [inline README.md]: Delete Line 2

**Comment by rodrigotiscareno** [comment]
I know you've tested this locally I assume with a PG database, but could it be possible to create a folder called "schemas" with the DDL of each table that you used? That way anyone can create the tables and test locally too :)

**Comment by rodrigotiscareno** [comment]
@Li-Nathaniel

**Comment by Li-Nathaniel** [comment]
I just made a bunch of commits as I fixed the appending issue in PSQL. The microservice now works correctly and the API endpoints have been developed. I have included comments for more clarity as well as the Schemas folder. The microservice is ready for demo. @rodrigotiscareno

**Pull request merged**